src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -343,7 +356,7 @@ public void removePredicate(String userName, String botName, String predicateNam
    */
   public void addToSet(String setName, String setValue) {
     // add to the set for the bot.
-    Bot bot = bots.get(getCurrentBotName().toLowerCase());
+    Bot bot = getBot(getCurrentBotName());

```

Comment:
You've marked getBot as depricated, but you're using it here.  perhaps it shouldn't be deprecated?

Reply:
removed deprecated - keeping private